backend/ipmi.pm

```diff
@@ -46,12 +46,19 @@ sub ipmitool {
     push(@cmd, split(/ /, $cmd));
 
     my ($stdin, $stdout, $stderr, $ret);
-    $ret = IPC::Run::run(\@cmd, \$stdin, \$stdout, \$stderr);
+    # Prevent failure due to occational self-tests/self-reboots of the IPMI controller
+    for (my $tries = 2; $tries >= 0; $tries--) {
```

Comment:
IMHO this can be written in simpler perl-style: `for (0 .. $tries)` as you do not use the variable within the loop. However I think there should be a debug output line reporting about the retry.